Fitness/Controllers/HomeController.swift

```diff
@@ -262,6 +262,11 @@ extension HomeController: UICollectionViewDelegate, UICollectionViewDelegateFlow
 
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
         if collectionView != mainCollectionView {
+            // MARK: - Fabric
+            Answers.logCustomEvent(withName: "Found Info on Homepage", customAttributes: [
+                "Section": sections[indexPath.section].rawValue
```

Comment:
Unsure, but this might be getting the wrong section, since `indexPath` is associated with `todaysClassesCollectionView` and not `mainCollectionView`, but the `sections` array is based on `mainCollectionView`

Reply:
just changed it, was definitely erroneous. how does it look now?